.ci/install_kata.sh

```diff
@@ -53,12 +54,18 @@ case "${KATA_HYPERVISOR}" in
 		"${cidir}/install_cloud_hypervisor.sh"
 		echo "Installing experimental_qemu to install virtiofsd"
 		install_qemu
+		if [[ "${arch}" == "x86_64" ]]; then
```

Comment:
Does that mean we get the Rust version for x86 and C for the other architectures?

Reply:
Yes, and I've already started the conversation with the maintainers of the other arches to add support for the rust version (see: https://katacontainers.slack.com/archives/C879ACQ00/p1652290987328349?thread_ts=1652288278.597539&cid=C879ACQ00).

Once the x86_64 version gets merged I will open issues with detailed instructions on how to proceed with the other arches.

Reply:
I could, but that's out of the scope of my work.

I will provide the needed info for others to build and test on the arches they support, but I am not committed to spend time on that.

Fair?